web-ui/src/components/volunteer/VolunteerRelationships.jsx

```diff
@@ -428,6 +479,38 @@ const VolunteerRelationships = ({ forceUpdate = () => {}, onlyMe = false }) => {
 
       {relationshipDialog()}
 
+      {/* New Organization Creation Dialog */}
+      <Dialog open={organizationDialogOpen} onClose={() => setOrganizationDialogOpen(false)}>
+        <DialogTitle>Create New Organization</DialogTitle>
+        <DialogContent>
+          <TextField
+            label="Name"
+            fullWidth
+            margin="dense"
+            value={newOrganization.name}
+            onChange={e => setNewOrganization({ ...newOrganization, name: e.target.value })}
+          />
+          <TextField
+            label="Description"
+            fullWidth
+            margin="dense"
+            value={newOrganization.description}
+            onChange={e => setNewOrganization({ ...newOrganization, description: e.target.value })}
+          />
+          <TextField
+            label="Website"
+            fullWidth
+            margin="dense"
+            value={newOrganization.website}
+            onChange={e => setNewOrganization({ ...newOrganization, website: e.target.value })}
+          />
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={() => setOrganizationDialogOpen(false)}>Cancel</Button>
+          <Button onClick={handleCreateNewOrganization}>Save</Button>
+        </DialogActions>
+      </Dialog>
+
```

Comment:
This dialog exists twice. I wonder if it could be extracted for reuse in both places?